c_api/src/taichi_vulkan_impl.cpp

```diff
@@ -236,9 +236,14 @@ void ti_export_vulkan_memory(TiRuntime runtime,
   VulkanRuntime *runtime2 = ((Runtime *)runtime)->as_vk();
   taichi::lang::DeviceAllocation devalloc = devmem2devalloc(*runtime2, memory);
   vkapi::IVkBuffer buffer = runtime2->get_vk().get_vkbuffer(devalloc);
+
+  auto tup = runtime2->get_vk().get_vkmemory_offset_size(devalloc);
+  VkDeviceMemory vk_mem = std::get<0>(tup);
```

Comment:
I'd prefer the structured binding syntax instead of this. `auto &[a, b, c] = ...`